modules/vnet_peering/variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "local_resource_group_name" {
+  description = "Name of the existing local peer resource group where to place the resources created."
+  type        = string
+}
+
+variable "peer_resource_group_name" {
+  description = "Name of the existing remote peer resource group where to place the resources created."
+  type        = string
+}
+
+variable "name_prefix" {
+  description = "Prefix name appended to the peering names."
+  default     = ""
+  type        = string
+}
+
+variable "local_vnet_name" {
+  description = "Local VNET name."
+  default     = ""
+  type        = string
+}
+
+variable "peer_vnet_name" {
+  description = "Peer VNET name."
+  default     = ""
+  type        = string
+}
+
+variable "local_allow_virtual_network_access" {
```

Comment:
this actually is applicable to all `local_` and `peer_` properties, this one and all the ones below: should we really duplicate them? This gives flexibility of course, but this is also a module that will be used with NGFWs, so traffic settings should be symetrical IMO.

Reply:
For `allow_virtual_network_access` and `allow_forwarded_traffic` they could be the same but the other two actually require combinations for the NVA to work - for example if you want to use `use_remote_gateways` in one VNET - you must have `use_remote_gateways` + `allow_gateway_transit` in the other one - otherwise it will not work .

It's a bit more variables - true - but at least it offers you full flexibility .